strings/src/main/res/values/strings.xml

```diff
@@ -1158,5 +1158,6 @@
     ##############################################
     -->
     <string name="open_settings">Open Settings</string>
+    <string name="entities_title">Entities</string>
```

Comment:
This should get a comment. Maybe something like "An entity is any person, place or thing that a form can be about. This title appears on an experimental settings screen to view entities managed in the current project." It may be a hard one to translate to some languages so the more detail the better. Maybe even link to the forum thread.